pints/tests/test_opt_optimisation_controller.py

```diff
@@ -44,14 +44,47 @@ def test_optimise(self):
         self.assertEqual(x.shape, (2, ))
         self.assertTrue(f < 1e-6)
 
+    def test_transform(self):
+        # Test optimisation with parameter transformation.
+
+        # Test with LogPDF
```

Comment:
Would these fail without a transformation? That'd be quite good, from a functional testing point of view :D 
(For example nelder-mead on logistic fails without a transform, and is very fast so good for testing)

Reply:
I would leave these out for another PR! :sweat_smile:

Reply:
Created a ticket for that #1200